src/config/theme.ts

```diff
@@ -74,6 +82,12 @@ export const baseThemeDef: ThemeOptions = {
     },
     alerts: {
       lightWarningBackground: '#fffde0',
+      highPriorityErrorBackground: '#FF7575',
```

Comment:
@gigxz a couple of questions about the colors that I wanted to discuss with you. You mentioned `lightWarningBackground` on slack, but I think it's not actually the same hex value as the one in the Figma files. A few of the other colors were present already in the theme -- e.g. the high priority border and body background were versions of the `theme.palette.error.main` with some transparency -- but most were hex values that weren't present yet anywhere in the codebase, as far as I could find. I'm not really sure what approach to take here with adding the colors -- naming them, where to put them, trying to consolidate similar colors? (e.g. the border color and the header background color for the high and medium priority alerts are slightly different values in Figma.) Let me know what you think, or if we should loop in Ash?

Reply:
> You mentioned lightWarningBackground on slack, but I think it's not actually the same hex value as the one in the Figma files

Good to know, we should update it in figma probably. We should use the existing value in the theme

> A few of the other colors were present already in the theme -- e.g. the high priority border and body background were versions of the theme.palette.error.main with some transparency -- but most were hex values that weren't present yet anywhere in the codebase, as far as I could find. I'm not really sure what approach to take here with adding the colors -- naming them, where to put them, trying to consolidate similar colors? (e.g. the border color and the header background color for the high and medium priority alerts are slightly different values in Figma.) Let me know what you think, or if we should loop in Ash?


Yeah good question, lets consult with Ash about the usage. For reference:

<img width="633" alt="Screenshot 2024-01-29 at 9 47 44 AM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/e66789f2-cca3-407e-ba73-7e0292ee20e2">
<img width="641" alt="Screenshot 2024-01-29 at 9 47 39 AM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/ef19ac6a-3efd-4d43-b55c-edd407afdd91">